setup.py

```diff
@@ -32,15 +45,14 @@
         "Programming Language :: Python :: 3.6",
         "Programming Language :: Python :: 3.7",
         "Programming Language :: Python :: 3.8",
-        "Programming Language :: Python :: Implementation :: CPython",
```

Comment:
Why was this line removed?

Reply:
Great question. We're not supporting any specific implementation, so I see no reason to have this line.

Reply:
Ahh. Understood.